core/src/main/java/io/opencensus/stats/AggregationData.java

```diff
@@ -130,10 +130,16 @@ static CountData create(long count) {
     HistogramData() {
     }
 
-    static HistogramData create(long... bucketCountDatas) {
-      checkNotNull(bucketCountDatas, "bucket counts should not be null.");
+    /**
+     * Create a {@code HistogramData}.
```

Comment:
The summary fragment should be an incomplete sentence, such as "`Creates a {@code HistogramData}.`" (https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment).  This applies to RangeData, too.